docs/5min-overview.md

```diff
@@ -72,15 +72,14 @@ Find our latest [node release on GitHub][node-release].
 Discover and contribute to our main projects [on GitHub][github]:
 
 - [Full node][full-node]
-- [Desktop wallet][desktop-wallet]
+- [Mobile Wallet & Desktop Wallet](https://github.com/alephium/alephium-frontend)
```

Comment:
I recommend making this change:

```suggestion
- [Desktop wallet][desktop-wallet]
- [Mobile wallet][mobile-wallet]
```

and then at the end of the file add the actual links:

```
[desktop-wallet]: https://github.com/alephium/alephium-frontend/apps/desktop-wallet
[mobile-wallet]: https://github.com/alephium/alephium-frontend/apps/mobile-wallet
```

@maud, this is a feature of markdown called "Link reference". It's useful when links need to be reused, so that you don't add the same link in many places in the same file, and it keeps all links grouped together so that it's easy to find when they need to be changed. You can read more [here](https://www.eddymens.com/blog/how-to-reuse-links-in-markdown-reference-links) if you want.